docs/NEW_GUIDE_DRAFT.en.md

```diff
@@ -0,0 +1,68 @@
+`NEAR CLI Core commands`
+login
+create-account
+    - implicit
+    - subaccount
+<!-- account Id is entered in this step -->
+manage-account
+    - keys
+        - view
+        - add
+        - delete
+        - nonce
+    - local-keys
+        <!-- Needs to be designed (with multikey management) -->
+    - contract
+        - deploy
+        - dev-deploy
```

Comment:
Hm.. it can be a part of development extension. What is your opinion @DiscRiskandBisque ?
And yes, it does not require an account Id, so it's out of place here.

Reply:
Moved to dev extension for now.

Reply:
I agree with frol. `dev-deploy` feels out of place with the other commands. Let's not include it here.

Even in the project extension it could be an option; something like `near project deploy --generateAccounts`.